api/v1/paths/games.js

```diff
@@ -22,8 +22,13 @@ const get = async (req, res) => {
  */
 const post = async (req, res) => {
   try {
-    const result = await gamesDao.postGame(req.body);
-    res.status(201).send(result);
+    const { developerId } = req.body.data.attributes;
+    if (!Number.isNaN(developerId) && !Number.isInteger(parseFloat(developerId))) {
+      errorBuilder(res, 400, ['developerId must be a string containing a valid integer']);
+    } else {
+      const result = await gamesDao.postGame(req.body);
+      res.status(201).send(result);
+    }
```

Comment:
If you wanna validate the input pattern, you should validate it by providing `pattern` keyword in your openapi file and have the express-openapi package handle it for you: Example: https://github.com/osu-mist/terms-api/blob/master/openapi.yaml#L39

Instead, you should put other additional logic checks here. For example, if I try to create a game bind with a non-existing developerId like:

```json
{
	"data": {
		"type": "game",
		"attributes": {
			"name": "test",
			"releaseDate": "2019-01-01",
			"developerId": "4"	
		}
	}
}
```

I still got a 500 since the parent key not found.

Reply:
Added pattern to all Resource definitions in openapi. Added method in games-dao to check if a developerId exists in the db.